src/applications/personalization/profile/tests/actions/directDeposit.unit.spec.js

```diff
@@ -0,0 +1,106 @@
+import sinon from 'sinon';
+import { expect } from 'chai';
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
+
+import {
+  fetchDirectDeposit,
+  toggleDirectDepositEdit,
+  fetchDirectDepositArgs,
+  DIRECT_DEPOSIT_FETCH_STARTED,
+  DIRECT_DEPOSIT_FETCH_SUCCEEDED,
+  DIRECT_DEPOSIT_FETCH_FAILED,
+  DIRECT_DEPOSIT_EDIT_TOGGLED,
+  DIRECT_DEPOSIT_API_ENDPOINT,
+} from '@@profile/actions/directDeposit';
+import environment from '~/platform/utilities/environment';
+import { base } from '../../mocks/endpoints/direct-deposits';
+
+import error500 from '../fixtures/500.json';
+
+describe('directDeposit actions', () => {
+  const endpointUrl = `${environment.API_URL}/v0${DIRECT_DEPOSIT_API_ENDPOINT}`;
+
+  let server = null;
+  let recordApiEventStub = null;
+  let captureErrorStub = null;
+
+  beforeEach(() => {
+    recordApiEventStub = sinon.stub(fetchDirectDepositArgs, 'recordApiEvent');
+    captureErrorStub = sinon.stub(fetchDirectDepositArgs, 'captureError');
+  });
+
+  afterEach(() => {
+    server.resetHandlers();
+    recordApiEventStub.restore();
+    captureErrorStub.restore();
+  });
+
+  after(() => {
+    server.close();
+  });
+
+  it('should dispatch DIRECT_DEPOSIT_INFORMATION_FETCH_SUCCEEDED with response on success', async () => {
```

Comment:
non-blocking: would it make more sense to rename this to the actual action name `DIRECT_DEPOSIT_FETCH_SUCCEEDED`?

Reply:
oh snap, yea you caught one one that I missed. I have INFORMATION in the original action names and didn't like it, so I removed it and didn't remove it from the test description. Thanks!